### PR TITLE
Note added Exchange Web Parts support

### PR DESCRIPTION
**Pull request by lusassl-msft**
Exchange 2016 / 2019 / EXO do not support Web Parts. Ping me internally for more information.

### CODE REVIEW

**Comment by lusassl-msft** [comment]
FYI: 
https://docs.microsoft.com/en-us/Exchange/new-features/discontinued-features?view=exchserver-2016#outlook-web-app-and-outlook

_Outlook on the web no longer includes support for Web Parts. Customers will need to develop replacement functionality to meet this need in their environments._

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeServer2013/using-outlook-web-app-web-parts-exchange-2013-help.md [hunk 1]
```diff
@@ -38,6 +38,9 @@ Depending on the security measures that have been applied to your network, you m
 
 The following table lists the parameters of a Web Part and examples of how they're used.
 
+> [!NOTE]
+> Exchange 2016 / 2019 and Exchange Online no longer includes support for Web Parts.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer2013/using-outlook-web-app-web-parts-exchange-2013-help.md:42]:
```suggestion
> Exchange 2016, Exchange 2019, and Exchange Online no longer include support for Web Parts.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: Exchange/ExchangeServer2013/using-outlook-web-app-web-parts-exchange-2013-help.md (hunk 1 above)
Comment (lusassl-msft) [reply Exchange/ExchangeServer2013/using-outlook-web-app-web-parts-exchange-2013-help.md:42]: Looks good. Let's do it as suggested.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**